modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/TestflowOutlet.tsx

```diff
@@ -43,6 +43,12 @@ const TestflowOutlet = () => {
 		],
 	});
 
+	useEffect(() => {
+		setHeaderActions({
+			actions: [],
+		});
+	}, [setHeaderActions]);
+
```

Comment:
@welitonpe I think this could be replaced by this property in useHeader params:

`useHeaderActions: {actions: []}`

Can you see if it works?

Reply:
ok